src/derivatives/atlas.md

```diff
@@ -196,17 +192,17 @@ A guide for using macros can be found at
                "tpl-MNIPediatricAsym_cohort-1_res-1_PD.nii.gz": "",
                "tpl-MNIPediatricAsym_cohort-1_res-1_T1w.nii.gz": "",
                "tpl-MNIPediatricAsym_cohort-1_res-1_T2w.nii.gz": "",
-               "tpl-MNIPediatricAsym_cohort-1_res-1_desc-brain_mask.nii.gz": "",
-               "tpl-MNIPediatricAsym_cohort-1_res-1_label-CSF_probseg.nii.gz": "",
-               "tpl-MNIPediatricAsym_cohort-1_res-1_label-GM_probseg.nii.gz": "",
-               "tpl-MNIPediatricAsym_cohort-1_res-1_label-WM_probseg.nii.gz": "",
+               "atlas-MNIPediatricAsym_cohort-1_res-1_desc-brain_mask.nii.gz": "",
+               "atlas-MNIPediatricAsym_cohort-1_res-1_seg-CSF_probseg.nii.gz": "",
+               "atlas-MNIPediatricAsym_cohort-1_res-1_seg-GM_probseg.nii.gz": "",
+               "atlas-MNIPediatricAsym_cohort-1_res-1_seg-WM_probseg.nii.gz": "",
```

Comment:
Ditto (thanks for catching that `desc-`  for that mask which is old and outdated)

```suggestion
               "tpl-MNIPediatricAsym_cohort-1_res-1_label-brain_mask.nii.gz": "",
               "tpl-MNIPediatricAsym_cohort-1_res-1_label-CSF_probseg.nii.gz": "",
               "tpl-MNIPediatricAsym_cohort-1_res-1_label-GM_probseg.nii.gz": "",
               "tpl-MNIPediatricAsym_cohort-1_res-1_label-WM_probseg.nii.gz": "",
```